webpack.config.js

```diff
@@ -0,0 +1,152 @@
+const path = require('path');
+
+const assetDir  = './core/';
+const outputDir = 'core/dist';
+const styleGuide = path.resolve( __dirname, './styleguide/');
+
+const npmPackage = './node_modules/';
+const devMode = process.env.NODE_ENV !== 'production';
+
+const webpack = require('webpack');
+const autoprefixer = require('autoprefixer');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const UglifyJsPlugin = require("uglifyjs-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
+const WebpackAssetsManifest = require("webpack-assets-manifest");
+const CopyPlugin = require('webpack-copy-on-build-plugin');
+
+// for MiniCssExtractPlugin
+function recursiveIssuer(module) {
+  if (module.issuer) {
+    return recursiveIssuer(module.issuer);
+  } else if (module.name) {
+    return module.name;
+  } else {
+    return false;
+  }
+}
+
+module.exports = {
+  entry: {
```

Comment:
```suggestion
  devtool: 'source-map',
  entry: {
```

Try this out to enable source maps along with enabling css sourcemaps in the CSS Loader.